src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -278,8 +279,18 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('thumbprint', nargs='+', help='A single or Space-separated list of client certificate thumbprint(s) to be remove.')
         c.argument('common_name', nargs='+', help='A single or Space-separated list of client certificate common name(s) to be remove.')
 
+    with self.argument_context('sf managed-cluster network-security-rule add', validator=validate_add_network_security_rule) as c:
+        c.argument('name', help='Network security rule name')
+        c.argument('access', help='possible values are <allow> or <deny>')
+        c.argument('direction', help='possible values are <inbound> or <outbound>')
```

Comment:
Could we define enumerations for these possible values by using `arg_type=get_enum_type()`?
```suggestion
        c.argument('access', help='possible values are <allow> or <deny>')
        c.argument('direction', help='possible values are <inbound> or <outbound>')
```

Reply:
In addition, please explain the function of these parameters in the help message

Reply:
Updated